cycle-scripts-es-browserify/scripts/init.js

```diff
@@ -4,14 +4,19 @@ var fs = require('fs-extra')
 var path = require('path')
 var spawn = require('cross-spawn')
 var chalk = require('chalk')
+var ejs = require('ejs')
 
-function dependencies (streamLib) {
-  var basicDependencies = [
-    '@cycle/dom',
-    'xstream'
-  ]
+function dependencies (libs) {
+  var basicDependencies = []
+  var core = {
+    cycle: ['@cycle/dom', 'xstream'],
+    motorcycle: ['@motorcycle/core', '@motorcycle/dom']
+  }
+  var extras = ['immutable', '@cycle/isolate']
```

Comment:
So, what's the point in installing immutable by default? `@cycle/isolate` seems to be a recurrent case, so, this could be included on all flavors in a proper PR, but immutable seems too opinionated to be in the core flavors.

Usually I prefer to use `ramda`, that is a totally different lib, but serves me for similar cases. But again, this is my preference, not the end-user.